resources/views/cruds/forms/attributes/_buttons.blade.php

```diff
@@ -37,7 +37,12 @@
         @endif
     </ul>
 </div>
-
+@if ($entity->attributes()->where('is_hidden', '0')->get()->has('0'))
```

Comment:
When creating a new entity, I have a crash because of this line. Same when editing an entity.